chaoslib/litmus/disk_fill/disk_fill_by_litmus.yml

```diff
@@ -73,23 +73,24 @@
         message: "Injecting {{ c_experiment }} chaos on {{ pod_name.stdout }}"
       when: "c_engine is defined and c_engine != ''"
 
-    - name: Create the DaemonSet for Disk-Fill
+    - name: Create the Job for Disk-Fill
       shell: >
-        kubectl apply -f /chaoslib/litmus/disk_fill/disk_fill_ds.yml -n {{ a_ns }}
+        kubectl apply -f /chaoslib/litmus/disk_fill/disk_fill_job.yml -n {{ a_ns }}
       args:
         executable: /bin/bash
-      register: disk_fill_ds_result
+      register: disk_fill_job_result
   
-    - name: Confirm that the disk-fill daemonSet is running on all nodes
+    - name: Confirm that the disk-fill job is running on {{ app_node }} node
```

Comment:
This and "Confirm that disk fill container is in running state " task both are doing the same thing so can you please remove this task?